src/app/sign-up/page.tsx

```
@@ -1,27 +1,81 @@
+      console.log("회원가입 실패", error);
+    }
+  };
+
+  if (errors) {
+    console.log("formError", errors);
+  }
+
   return (
     <div className="flex h-screen w-full items-center justify-center">
       <div className="h-full w-[50%] bg-gradient-to-r from-[#3A57E8] to-[#213182]"></div>
       <div className="flex h-full w-[50%] items-center justify-center">
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)